lib/ws-browser.js

```diff
@@ -8,6 +8,54 @@ const mdsf = require('mdsf');
 const constants = require('./internal-constants');
 const transportCommon = require('./transport-common');
 
+const SEND_INTERVAL = 50;
+const BUFFER_SIZE_LIMIT = 64000;
+
+const utf16singleUnit = 1 << 16;
```

Comment:
I don't think it will simplify anything, we will still have to convert the strings that we receive from `mdsf.stringify()` to binary UTF-8 representation.

Reply:
Yeah but we have Buffer polyfill anyway, so it's a one-liner.

Reply:
Using binary frames will also disallow inspecting the WebSocket frames in browser's dev tools and won't make it easy to split UTF-8 messages if we want to avoid the possibility of breaking a single Unicode character between frames.
I am not sure if these are important to consider, though.